ui/app/templates/components/shamir-flow.hbs

```diff
@@ -23,20 +19,19 @@
           <code class="is-word-break">{{otp}}</code>
         </div>
       </div>
+    {{/if}}
       <div class="message is-list has-copy-button" tabindex="-1">
-        {{#let
-            (concat 'vault operator generate-root -dr-token -otp="' otp '" -decode="' encoded_token '"')
-        as |cmd|}}
+        {{#with (if otp
```

Comment:
I read the documentation on these template helpers, but I'm unclear why it's better to use "with" here instead of "let".  This is more of a clarity question as I'm not super familiar with these template helpers.

Reply:
this is an excellent question! your instincts to use `let` were spot on, as`let` is increasingly preferred over `with`. the difference is that `with` allows you to only render a block if the first positional param is true. that's useful in this situation, but we can easily accomplish the same outcome by using `let` and `if`. the example in this RFC which calls for deprecating `with` explains this really well:

https://github.com/emberjs/rfcs/blob/master/text/0445-deprecate-with.md#motivation